charts/spire/charts/spire-server/values.yaml

```diff
@@ -20,6 +20,9 @@ image:
   pullPolicy: IfNotPresent
   tag: ""
 
+## @param deploymentType Define SPIRE server deployment type. Can be statefulset/deployment. Defaults to statefulset if not set.
```

Comment:
This feature should be marked in the comment as experimental explicitly so if folks are just looking at values, they will see it.

Reply:
Updated.